install/install.sh

```diff
@@ -93,7 +93,7 @@ checkJqInstalled() {
 }
 
 getLatestRelease() {
-    local spiceReleaseUrl="https://api.github.com/repos/${GITHUB_ORG}/${GITHUB_REPO}/releases"
+    local spiceReleaseUrl="https://api.github.com/repos/${GITHUB_ORG}/${GITHUB_REPO}/releases/latest"
     local latest_release=""
```

Comment:
Does the logic below to set `latest_release` still work with this change? `/releases` returns an array and `/releases/latest` returns just an object.

Reply:
Yes, I've checked that - code below just extracts first `tag_name` with assertions from the output, it doesn't require it to be an array

Example:

![CleanShot 2024-05-06 at 11 24 26@2x](https://github.com/spiceai/spiceai/assets/827338/17ba1aa9-b7ed-4304-a730-2cd124833a2a)

Reply:
For spiceai repo

![CleanShot 2024-05-06 at 11 29 08@2x](https://github.com/spiceai/spiceai/assets/827338/a0d8fdc8-67aa-4a73-9598-2c47f4938367)